graylog2-web-interface/src/views/logic/views/ViewTransformer.js

```diff
@@ -30,11 +30,17 @@ const ViewTransformer = (searchView: View): View => {
       .widgets(widgets)
       .build();
   });
+  // Remove query string attached to the existing search query
+  const newQueries = searchView.search.queries.map(
+    query => query.toBuilder().query({ ...query.query, query_string: '' }).build(),
+  );
+  const newSearch = searchView.search.toBuilder().queries(newQueries).build();
```

Comment:
With these changes the `search -> query` equals the query which gets created, on a dashboard creation. Which means it still has its own timerange. I thought about removing the timerange as well, but it is a required attribute and because every widget has its own timerange, this will not result in any problems.